src/hooks/feed/use-book-feed-utils.ts

```diff
@@ -45,8 +46,17 @@ export function calculateBookStats(books: {
   const pagesRead = books.read.reduce((sum, book) => sum + (book.pageCount || 0), 0);
   
   const ratedBooks = books.read.filter(book => book.readingStatus?.rating !== undefined);
+  
+  // Convert each Rating to a fraction value before summing
+  const ratingSum = ratedBooks.length > 0
+    ? ratedBooks.reduce((sum, book) => {
+        const rating = book.readingStatus?.rating;
+        return sum + (rating instanceof Rating ? rating.fraction : 0);
```

Comment:
I think TypeScript should enforce that the rating variable is always a Rating object here, and if not then the type should be fixed wherever it's coming from :thinking: 

```suggestion
        return sum + rating.fraction;
```